### ISSUE TITLE
Dialog vs. AlertDialog

### ISSUE THREAD

**Issue by benbeaudry** [w3c/aria#1799]
What's the actual difference between a [dialog](https://w3c.github.io/aria/#dialog) and a [alertdialog](https://w3c.github.io/aria/#alertdialog)?

An alertdialog is a subclass of dialog and should be modal, just like a dialog. Both roles expect the focus to be within the dialog. The only apparent difference is that an alertdialog should convey a message to alert the user. In practice, though, I think screen readers will just read out the content of dialogs even though there is no alert.

Can someone give me specific use cases for each?

**Comment by aleventhal** [comment]
Agree. It seems like any possible difference will be really minor.

I see this difference:
* An alertdialog fires an alert event, so that the screen reader should read the entire thing.
* A dialog does not, and the screen reader should read up to the first focus (hopefully all ATs do this).
(In both cases, if aria-describedby/aria-description is set on them, then that should be read instead.)

If there are no other practical differences for the user, then perhaps we should deprecate alert dialog.

**Comment by scottaohara** [comment]
another potential difference would be a `role=dialog` should be considered `aria-modal=false` by default, where as an `alertdialog` could be `aria-modal=true` by default.  

https://github.com/w3c/aria/issues/1210

**Comment by aleventhal** [comment]
Weird, but ok. I would have thought the opposite maybe? :)

**Comment by scottaohara** [comment]
@aleventhal maybe something else for us to talk out then.  I'm curious as to why you'd think the opposite.

**Comment by JAWS-test** [comment]
I guess the main difference is the behavior when the browser window doesn't have the focus.
* When a dialog is opened, something just happens on the browser window in the background and this is not noticeable with AT. 
* if an alertdialog is opened, then an alert-message is sent, which makes the operating system output this message from the AT, even if the browser window is invisible in the background.

**Comment by aleventhal** [comment]
@JAWS-test thanks, that's interesting.
@scottaohara we should test this to verify it is correct.
